Sources/PrimerSDK/Classes/User Interface/TestPaymentMethods/FlowDecisionTableViewCell.swift

```diff
@@ -0,0 +1,70 @@
+//
+//  FlowDecisionTableViewCell.swift
+//  PrimerSDK
+//
+//  Created by Dario Carlomagno on 29/05/22.
+//
+
+#if canImport(UIKit)
+
+import UIKit
+
+class FlowDecisionTableViewCell: UITableViewCell {
+    
+    static var identifier: String = "FlowDecisionTableViewCell"
+        
+    internal private(set) var decision: PrimerTestPaymentMethodOptions.FlowDecision!
+    private let cellInternalPadding: CGFloat = 4
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        
+        self.preservesSuperviewLayoutMargins = false
+        self.contentView.preservesSuperviewLayoutMargins = false
+        self.selectionStyle = .none
+        
+        let theme: PrimerThemeProtocol = DependencyContainer.resolve()
+        backgroundColor = theme.view.backgroundColor
+        layer.cornerRadius = 8
```

Comment:
I believe that the current structure of theme contains PMs corner radius

Reply:
That corner radius is for the decision cells "Authorized / Declined / Fail". PM corner radius hasn't been touched 👍